d9/providers/githubactions/.ci/.github/workflows/build_deploy_and_test.yml

```diff
@@ -47,29 +47,23 @@ jobs:
       # https://github.com/pantheon-systems/docker-build-tools-ci/blob/6.x/scripts/set-environment
       - name: setup-environment-vars
         run: |
-          if [ "$CI_BRANCH" != "refs/heads/master" ]; then
+          # Test vars not set means trigger = push.
+          if [ -z "$PR_NUMBER" ]; then
             export PR_NUMBER=`curl -u "${GITHUB_OWNER}:${GITHUB_TOKEN}" \
```

Comment:
This should not run at all when trigger is pull_request as the right values should be already set at this point